README.md

```diff
@@ -184,6 +184,10 @@ optional arguments:
                         Hail version to use (default: 0.1).
   --master-machine-type MASTER_MACHINE_TYPE, --master MASTER_MACHINE_TYPE, -m MASTER_MACHINE_TYPE
                         Master machine type (default: n1-highmem-8).
+  --master-memory-fraction MASTER_MEMORY_FRACTION
+                        Fraction of master memory allocated to the JVM. Use a
+                        smaller number to reserve more more memory to Python.
```

Comment:
Fraction of master memory allocated to the JVM. Use a
smaller value to reserve more memory for Python.